src/main/kotlin/com/autonomousapps/internal/advice/AdvicePrinter.kt

```diff
@@ -6,7 +6,8 @@ import org.gradle.api.GradleException
 /**
  * Only concerned with human-readable advice meant to be printed to the console.
  */
-internal class AdvicePrinter(private val computedAdvice: ComputedAdvice) {
+internal class AdvicePrinter(private val computedAdvice: ComputedAdvice,
```

Comment:
I'm gonna nitpick stuff like this because I dislike this convention. If the line is too long, then it should be
```
class Thing(
  private val one,
  private val two
)
```

Reply:
we should use ktlint...

Reply:
Maybe. I do have an `.editorconfig`, but I suppose it doesn't catch this.